supplementary/devices/index.rst

```diff
@@ -452,6 +453,25 @@ Devices
      - Navigation
      - Sensors
      - Other
+   * - :doc:`Sony Xperia L <taoshan>`
+     - taoshan
+     - not ported yet
+     - Halium Kernel
```

Comment:
there is no such thing as "Halium Kernel". You should indicate a Linux Kernel version used in LineageOS for your device.

Reply:
I meant the linux kernel which halium is based on. I filled in the version.